themes/ddbasic/sass/components/ting-object/ting-object.scss

```diff
@@ -0,0 +1,441 @@
+@import '../../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?